test/integration/features/addon-xpaas.feature

```diff
@@ -19,7 +19,8 @@ which are then available in OpenShift to the user.
   Scenario Outline: User deploys, checks out and deletes several templates from XpaaS imagestream
    Given Minishift has state "Running"
     When executing "oc new-app <template-name>" succeeds
-     And services "<service-name>" rollout successfully
+     And service "<service-name>" rollout successfully
+     And user waits "5" seconds
```

Comment:
why do you need a sleep? How do you know that sleeping this time interval will work? we explicitly avoided to use sleep.